site/src/components/gradients.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Gradients
+description: Gradient related components in the Atlas Design System
+template: standard
+---
+
+# Gradients
+
+There are two main types of gradients.
+
+1. Gradients for blending colors between elements and contains (`.graident-border-*`).
+2. Gradients displayed with font. Coming soon.
+
+## Gradient borders usage
+
+The `.gradient-border` component is a moderately pluggable component used to blend the boundaries between colors. Because of the near-infinite number of steps, color variations, and gradient types involved in creating a gradient. We've scope these to be linear gradients that transition between `transparent` and a given color represented by the `--border-gradient-end-color` custom property. Each gradient combination requires a minimum of two class, one specifying the color and one specifying the direcition.
```

Comment:
nit: grammar and spelling
```suggestion
The `.gradient-border` component is a moderately pluggable component used to blend the boundaries between colors. Because of the near-infinite number of steps, color variations, and gradient types involved in creating a gradient, we've scoped these to be linear gradients that transition between `transparent` and a given color represented by the `--border-gradient-end-color` custom property. Each gradient combination requires a minimum of two class, one specifying the color and one specifying the direction.
```

Reply:
updated!